ccd-definition/CaseEvent/MultiState.json

```diff
@@ -47,8 +49,9 @@
     "CaseTypeID": "CARE_SUPERVISION_EPO",
     "ID": "allocatedJudge",
     "Name": "Allocated Judge",
+    "DisplayOrder": 2,
     "Description": "Add allocated judge to a case",
-    "PreConditionState(s)": "Submitted;Gatekeeping;PREPARE_FOR_HEARING",
+    "PreConditionState(s)": "Gatekeeping;PREPARE_FOR_HEARING",
```

Comment:
Is this correct? AC says admin should have permissions for: Allocated judge (Submitted, Gatekeeping, Prepare for hearing)
I don't see details in the ticket to remove this from submitted (note that you can't create any hearings without an allocated judge), but maybe the discussion happened elsewhere

Reply:
Yes this changed and was agreed with Gail, Jay, Jason - discussions happened and ticket wasn't updated so i guess this was just missed out of the screenshots I added of things changed. but good point about not being able to add any hearings without it so will mention this to gail to see if it changes anything.

Reply:
resolving this - i've raised it with gail and she is checking with jason jay shanaz but she said she can create another story to resolve this workflow if it's required